reference/promise-types/reports.markdown

```diff
@@ -6,19 +6,18 @@ tags: [reference, bundle common, reports, promises]
 ---
 
 Reports promises simply print messages. Outputting a message without
-qualification can be a dangerous operation. In a large installation it could
-unleash an avalanche of messaging.
+qualification can be a dangerous operation. In a large installation it
+could unleash an avalanche of messaging, so it is reccomended that reports are
+appropriately.
```

Comment:
`s/reccomended/recommended/`. Also the last sentence ends abruptly, and doesn't make sense.